markups/cmark_render.py

```diff
@@ -0,0 +1,38 @@
+# vim: ts=8:sts=8:sw=8:noexpandtab
+
+# This file is part of python-markups module
+# License: BSD
+# Copyright: (C) Shengjing Zhu, 2016
+
+from CommonMark import HtmlRenderer
+from CommonMark.common import escape_xml
+
+
+class CommonMarkRender(HtmlRenderer):
```

Comment:
Also, can this class be moved to `commonmark.py`, preferably with a leading underscore to make it private?